src/ios/Thumbnail.m

```diff
@@ -68,15 +68,19 @@ + (NSString *) uuid {
 
 @implementation Thumbnail
 
-+ (void) thumbnail:(NSString *)imageURL size:(CGFloat)maxSize toURL:(NSString *) toURL compression:(CGFloat) compressionQuality
++ (void) thumbnail:(NSString *)imageURL size:(CGFloat)maxSize toURL:(NSString *) toURL compression:(CGFloat) compressionQuality outputFormat:(NSString *) outputFormat
 {
 
     NSURL* _imageURL = [NSURL URLWithString: imageURL];
 
     UIImage *uiImage = [self thumbnailWithContentsOfURL:_imageURL maxPixelSize:maxSize];
     if(uiImage) {
         NSError *writeError = nil;
-        [UIImageJPEGRepresentation(uiImage, compressionQuality) writeToFile:toURL options:NSDataWritingAtomic error:&writeError];
+        if([outputFormat isEqualToString:@"PNG"]){
```

Comment:
What about WEBP? Since we have the option above...

Reply:
I couldn't find any native support for it, unfortunately. It looks like there are some third party options for including, but I decided it wasn't worth it. I'd be more inclined to remove the WEBP support from the android version and make all three platforms consistent.

Reply:
Okay, perhaps an error message if it's asked for?